package.json

```diff
@@ -36,11 +36,15 @@
   },
   "devDependencies": {
     "@zazuko/rdf-vocabularies": "^2019.10.22",
+    "@tpluscode/rdf-string": "^0.2.7",
+    "@types/sinon": "^7.5.2",
```

Comment:
Are these deps required?

Reply:
Ah, yes, `rdf-string` is not used. Must have installed to inspect something but shouldn't have added to package.json

The other one? Well, I don't have the entire Sinon memorized. `@types` greatly improves the IDE suggestions in webstorm

Reply:
@bergos